dtfs-central-api/src/v1/swagger-definitions/utilisation-reports/utilisation-report-data-validation.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @openapi
+ * definitions:
+ *   RawReportDataWithCellLocations:
+ *     type: array
+ *     items:
+ *       type: object
+ *       additionalProperties:
+ *         type: object
+ *         properties:
+ *           value:
+ *             type: string
+ *             nullable: true
+ *           column:
+ *             type: string
+ *           row:
+ *             oneOf:
+ *               - type: string
+ *               - type: number
```

Comment:
Haven't seen this syntax before. Have we checked that it works?

Reply:
Yes, it's here in the docs: https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/
![Screenshot 2024-08-30 at 08 46 01](https://github.com/user-attachments/assets/c5a5e774-19ed-487c-9800-f286ff983285)